scripts/swupd-performance/stagingfilegen.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+
+echo "OUTFILE=$HOME/staging/file_"$(date +%Y%m%d_%H%M%SZ) > /etc/swupd_perform_update_config/script_config
```

Comment:
you have some shell violations here too, shellcheck will tell you

Reply:
looks like i fixed this